docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -112,12 +148,56 @@ Move the modified `node-config.yaml` to your Kubernetes node machine and install
 sudo coreos-install -d /dev/sda -C stable -c node-config.yaml
 ```
 
-Once complete, eject the bootable disc and restart the server.  When it comes back up, you should have SSH access as the `core` user using the public key provided in the `node-config.yaml` file.  It will take some time for the node to be fully configured.  Once fully configured, you can check that the node is running with the following command on the Kubernetes master.
+Once complete, eject the bootable disc and restart the server.  When it comes back up, you should have SSH access as the `core` user using the public key provided in the `node-config.yaml` file.  It will take some time for the node to be fully configured.
+
+## Configure Kubeconfig
+To administrate your cluster from a separate host, you will need the client and admin certificates generated earlier (`ca.pem`, `admin.pem`, `admin-key.pem`). With certificates in place, run the following commands with the appropriate filepaths.
+```
+kubectl config set-cluster calico-cluster --server=https://<KUBERNETES_MASTER> --certificate-authority=<CA_CERT_PATH>
+kubectl config set-credentials calico-admin --certificate-authority=<CA_CERT_PATH> --client-key=<ADMIN_KEY_PATH> --client-certificate=<ADMIN_CERT_PATH>
+kubectl config set-context calico --cluster=calico-cluster --user=calico-admin
+kubectl config use-context calico
+```
+
+Check your work with `kubectl get nodes`.
+
+## Install the DNS Addon
+
+Most Kubernetes deployments will require the DNS addon for service discovery. The config repository for this guide comes with manifest files to start the DNS addon.  To install DNS, create the skydns service and replication  controller found in the repo. 
+
+`kubectl create -f calico-kubernetes/config/master/dns/skydns.yaml`
```

Comment:
If you're just grabbing single files from the repo (as recommended above) then this line will need changing too.